packages/tldraw/src/lib/ui.css

```diff
@@ -38,6 +38,7 @@
 	font-size: 12px;
 	gap: 0px;
 	color: var(--color-text-1);
+	z-index: 0;
```

Comment:
prbly this can be omitted fwiw, but meh, sometimes it's nice to be explicit about it to show that there's an interaction later on